lib/interviewer/containers/Canvas/PresetSwitcherKey.js

```diff
@@ -125,13 +129,18 @@ const makeMapStateToProps = () => {
 
   const mapStateToProps = (state, props) => {
     const highlightLabels = get(props, 'preset.highlight', []).map((variable) =>
-      getNodeAttributeLabel(state, { variableId: variable, ...props }),
+      getNodeAttributeLabel(state, {
+        stage: getCurrentStage(state),
```

Comment:
given that this is happening inside a `map` it would be good to just calculate the current stage once, and then use that variable inside of the map. You can just to `const stage = getCurrentState(state)` in side of `mapStateToProps`, and then reuse it.

Reply:
resolved [a44327c](https://github.com/complexdatacollective/Fresco/pull/77/commits/a44327c33624fcce25e17666f987c8989ff2f1fa)